airflow/contrib/utils/sendgrid.py

```diff
@@ -30,8 +33,8 @@
 from airflow.utils.log.logging_mixin import LoggingMixin
 
 
-def send_email(to, subject, html_content, files=None, dryrun=False, cc=None,
-               bcc=None, mime_subtype='mixed', sandbox_mode=False, **kwargs):
```

Comment:
These changes don't look right at first glance. What's going on here?

Reply:
The removed function params are never used in the function